app/scripts/services/dimVendorService.factory.js

```diff
@@ -314,7 +314,8 @@
                       throw new Error("Cached failed vendor " + vendorDef.summary.vendorName);
                     });
                 }
-                throw new Error("Failed to load vendor " + vendorDef.summary.vendorName);
+                return $q.reject();
+                // throw new Error("Failed to load vendor " + vendorDef.summary.vendorName);
```

Comment:
Let's put this back and instead flip the vendor feature flag using your isExtension check.